docs/encryption/README.md

```diff
@@ -0,0 +1,27 @@
+# Encryption
+
+TODO: add some basics docs about how is encryption implemented/handled/used.
+
+## Lucene
+
+### Updating Lucene
+
+Identify what version of lucene is used by this elasticsearch
+
+    grep lucene buildSrc/version.properties
+
+Example output:
+
+    lucene            = 8.2.0
+
+To update the encrypted lucene you need to package the encrypted version of proper lucene version (e.g. `8.2.0`) and copy the archive into `lucene/libs`:
+
+    cd [path/to/lucene-solr]/lucene
+    git co branch_8_2_0_encrypted
+    ant package
```

Comment:
There is no target `package`. I used `ant jar` to build it. Maybe different version of ant? Mine is 1.10.7-1.

Reply:
There is no target `package` in `lucene-solr/lucene/core` (there is in `lucene-solr/lucene`) but you are right - since we want only `core` it is best to just `jar` the `core` as opposed to `package` the whole lucene